source/nodejs/adaptivecards-designer/src/app.ts

```diff
@@ -428,17 +454,6 @@ class DesignerApp {
         return this._hostContainerPicker;
     }
 
-    newCard() {
```

Comment:
Why remove this? The "New card" button in the toolbar and this newCard method are part of the app, please revert.

Reply:
P0: new card should be there - or at least don't remove the new card that was already there.  from a UX perspective, this does not handle protecting user from deleting what they already had, and that's OK for now.  we can add that in later.